streams/src/main/java/org/apache/kafka/streams/processor/internals/Task.java

```diff
@@ -51,7 +51,7 @@
      *          |            |                    |   |       |
      *          |            v                    v   |       |
      *          |      +-----+-------+       +----+---+----+  |
-     *          |      | Running (2) | ----> | Suspended(3)|  |    * //TODO Suspended(3) could be removed after we've stable on KIP-429
+     *          |      | Running (2) | ----> | Suspended(3)|  |    //TODO Suspended(3) could be removed after we've stable on KIP-429
```

Comment:
When we could actually close this TODO btw? @guozhangwang

Reply:
When we can remove the `eager protocol` I think.